kedro-org/kedro-website
# Feature/use logos layout for companies using kedro

## Description
Fixes #58 

## Development notes

I've updated the new layout designed by @Mackay031 (with the screenshot below)

<img width="899" alt="Screenshot 2022-09-21 at 12 42 54" src="https://user-images.githubusercontent.com/32060364/191495385-d51c1b83-ecf5-429f-9231-740c7c3bd8aa.png">

## QA notes

it has been tested manually by me to ensure all the logos are in the right order and position (with design guidelines as below)
<img width="881" alt="logos in place" src="https://user-images.githubusercontent.com/32060364/191495193-2067e440-c0c0-4453-b595-7d36ca94af46.png">


## Checklist
